Library/ViewModels/PledgePaymentPlansOptionViewModel.swift

```diff
@@ -1,13 +1,34 @@
+import Foundation
+import KsApi
 import ReactiveSwift
 
 public typealias PledgePaymentPlanOptionData = (
   type: PledgePaymentPlansType,
-  selectedType: PledgePaymentPlansType
+  selectedType: PledgePaymentPlansType,
+  paymentIncrements: [PledgePaymentIncrement], // TODO: replece with API model
```

Comment:
```suggestion
  paymentIncrements: [PledgePaymentIncrement], // TODO: replece with API model in [mbl-1838](https://kickstarter.atlassian.net/browse/MBL-1838)
```

Reply:
Updated, thanks for adding the ticket ref